config.py

```diff
@@ -55,6 +55,7 @@ class BaseConfig(object):
     #     raise ValueError('You need to export SECRET_KEY set for Flask application')
 
     SECURITY_PASSWORD_SALT = os.getenv("SECURITY_PASSWORD_SALT")
+    RESET_PASSWORD_SALT = os.getenv("RESET_PASSWORD_SALT")
```

Comment:
Why do we need this extra environment variable? Can you please explain that?
Assuming this is needed, can you please add this new env variable to documentation in `docs/environment_variables.md`

Reply:
I have seen the documentation of configuration where under miscellaneous there is a salt called --> **SECURITY_RESET_SALT** which Specifies the salt value when generating a password reset links/tokens. also, i have modified the name of **RESET_PASSWORD_SALT** back to **SECURITY_RESET_SALT** to remove confusion